lib/timeline/component/ItemSet.js

```diff
@@ -1143,7 +1140,7 @@ class ItemSet extends Component {
     const me = this;
 
     ids.forEach(id => {
-      const itemData = me.itemsData.get(id, me.itemOptions);
```

Comment:
Why was `me.itemOptions` removed?

Reply:
I found no place where it would be altered and it just set the same type coercion as was used everywhere anyway.